cypress/features/regression/experimental/checkbox.feature

```diff
@@ -1,88 +1,84 @@
 Feature: Experimental Checkbox component
-  I want to change Experimental Checkbox properties
-
-  Background: Open Experimental Checkbox component page
-    Given I open "Experimental Checkbox" component page
+  I want to check Experimental Checkbox properties
 
   @positive
   Scenario Outline: Change Checkbox component label to <label>
-    When I set label to <label> word
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "<nameOfObject>" object name
     Then checkbox label on preview is <label>
     Examples:
-      | label                        |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | label                   | nameOfObject          |
+      | mp150ú¿¡üßä             | labelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | labelSpecialCharacter |
 
   @positive
   Scenario: Disable and enable checkbox
-    Given I check disabled checkbox
-    When I uncheck disabled checkbox
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "enabledCheckbox" object name
```

Comment:
could you make change and use naming like `knobName+Parameter`, we have dedicated `json` for every component so `disabledFalse` will be enough.
```suggestion
    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "disabledFalse" object name
```

Reply:
fixed